<launch>.
 
  <!-- Initial pose right out of the entrance of vibot plateau -->
  <param name="initial_pose_x" type="double" value="3.5"/>
  <param name="initial_pose_y" type="double" value="1.47"/>
  <param name="initial_pose_a" type="double" value="0.01"/>
  
  <arg name="initial_pose_x" default="3.5"/>
  <arg name="initial_pose_y" default="1.47"/>
  <arg name="initial_pose_a" default="0.01"/>
  
  
  
  <param name="center_pose_x" type="double" value="-2.8"/>
  <param name="center_pose_y" type="double" value="-1.20"/>
  <param name="center_pose_a" type="double" value="0.0"/>
  
  <arg name="center_pose_x" default="-2.8"/>
  <arg name="center_pose_y" default="-1.20"/>
  <arg name="center_pose_a" default="0.0"/>  
  
    
  <param name="pick_pose_x" type="double" value="-2.61"/>
  <param name="pick_pose_y" type="double" value="-2.8"/>
  <param name="pick_pose_a" type="double" value="0.0"/>
  
  <arg name="pick_pose_x" default="-2.61"/>
  <arg name="pick_pose_y" default="-2.8"/>
  <arg name="pick_pose_a" default="0.0"/>  

  <!-- Map server--> 
   


  <!-- include amcl demo of turtlebot_gazebo -->
  <!-- include file = "$(find turtlebot_navigation)/launch/amcl_demo.launch" -->
 <include file = "$(find solid_navigation)/launch/amcl_demo.launch">
  <!--arg name = "map_file" value="~/ros/indigo/catkin_ws/src/solid_navigation/maps/map.yaml" / -->
  <!-- use the initial pose coordinates above -->
  <rosparam command="delete" ns="move_base" /> 
  <arg name="initial_pose_x" value="$(arg initial_pose_x)"/> 
  <arg name="initial_pose_y" value="$(arg initial_pose_y)"/> 
  <arg name="initial_pose_a" value="$(arg initial_pose_a)"/> 
  <arg name="map_file" default="/home/turtlebot/ros/indigo/catkin_ws/src/solid_navigation/maps/map.yaml"/> 
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" > 
  </node> 
  </include>  


  
 <!-- launch file to open rviz -->
  <!-- bringup rviz -->
  <!-- include file = "$(find turtlebot_rviz_launchers)/launch/view_navigation.launch" -->
 <!--  </include> -->

</launch>
